src/Spatial/Euclidean/ILine.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace MathNet.Spatial.Euclidean
+{
+    /// <summary>
+    /// This interface represents a line in cartesian space
+    /// </summary>
+    public interface ILine : IEquatable<ILine>
```

Comment:
Name it `ILine2D` until we have decided on namespaces.

Reply:
ok